fixes #1243 - Add refrences to alt with no value

scottaohara opened the pull request:
This commit adds prose to reference that an `alt=“”` and `alt` with no set value will both be interpreted as an empty string, per [issue #1243](https://github.com/w3c/html/issues/1243).

Happy to hear any/all feedback.  Thanks!

scottaohara committed:
add refrences to alt with no value

This commit adds prose to reference that an `alt=“”` and `alt` with no set value will both be interpreted as an empty string.

scottaohara commented:
On a related note, I am separately working through [#885](https://github.com/w3c/html/issues/885). And no changes to that particular part of the spec are in this pull request.

edent commented on sections/semantics-embedded-content.include:
```diff
@@ -2981,56 +2981,65 @@
-  When images are used inline as part of the flow of text in a sentence, provide a word or phrase as a text
-  alternative which makes sense in the context of the sentence it is apart of.
+  When images are used inline as part of the flow of text in a sentence, provide a word or
+  phrase as a text alternative which makes sense in the context of the sentence it is apart of.
```
Should be "a part of" not "apart"

scottaohara committed:
fix apart typo

chaals commented on sections/semantics-embedded-content.include:
```diff
@@ -2981,56 +2981,65 @@
   In many cases, the image is actually just supplementary, and its presence merely reinforces the
   surrounding text. In these cases, the <code>alt</code> attribute must be
-  present but its value must be the empty string.
+  present but its value must be unset or an empty string.
```
"unset or the empty string" is probably more consistent with how we say it elsewhere.

chaals reviewed (commented):
tiny editorial comment. otherwise looks OK.

Do we want to be coherent in our usage of `alt=""` or just `alt`?

scottaohara committed:
“or the empty string”

scottaohara committed:
fix typos

This commit contains the following fixes:

- two instances of “relevant” right next to each other, clearly should be one.

- typo of “alternatve” should be “alternative”

chaals reviewed (approved):
ok, looks good.
